desktop/install/linux-install.md

```diff
@@ -268,6 +270,47 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the
+default (and currently only) mechanism to enable file sharing between the host
+and Docker Desktop VM. In order not to require elevated privileges, without
+unnecessarily restricting operations on the shared files, Docker Desktop runs
+the file sharing service (`virtiofsd`) inside a user namespace (see
+`user_namespaces(7)`) with UID and GID mapping configured. As a result Docker
+Desktop relies on the host being configured to enable the current user to use
+subordinate ID delegation. For this to be true `/etc/subuid` (see `subuid(5)`)
+and `/etc/subgid` (see `subgid(5)`) have to be present. Docker Desktop only
+supports subordinate ID delegation configured via files. In case of both,
+`/etc/subuid` and `/etc/subgid`, Docker Desktop uses the first entry
+corresponding to the current user, to set up mappings for IDs above 0, and map
+the current users ID and GID to 0.
+
+| ID in container | ID on host                                                                       |
+| --------------- | -------------------------------------------------------------------------------- |
+| 0 (root)        | ID of the user running DD (e.g. 1000)                                            |
+| 1               | 0 + beginning of ID range specified in `/etc/subuid`/`/etc/subgid` (e.g. 100000) |
+| 2               | 1 + beginning of ID range specified in `/etc/subuid`/`/etc/subgid` (e.g. 100001) |
+| 3               | 2 + beginning of ID range specified in `/etc/subuid`/`/etc/subgid` (e.g. 100002) |
+| ...             | ...                                                                              |
+
+If `/etc/subuid` and `/etc/subgid` are missing, they will need to be created.
+Both should contain entries in the form -
+`<username>:<start of id range>:<id range size>`. E.g. to allow a user whose
+username is `myusername` to use IDs from 100000 to 165535:
+
+```console
+$ echo 'myusername:100000:65536' | sudo tee /etc/subuid
+$ echo 'myusername:100000:65536' | sudo tee /etc/subgid
```

Comment:
I am on the fence about this one, since it makes the user even less likely to read the commands and change what they need i.e. substitute `myusername` with the username they need to configure.

Reply:
What about "$USER", the thing is that tee without `-a` will overwrite the file.

Reply:
Sure, that works :)

Reply:
Question is; do we _want_ these to be appended? The risk there is that someone runs it twice and ends up with a configuration they don't want. The instructions should at least contain a validation step afterwards to check permissions of the file and its content.

Reply:
That's a good question. On one hand the instruction is for when the files are missing, so overwriting would be ok. But were the user not to read that we do not want to overwrite an existing config. So, Fredo's original suggestion would work better. I do say above that we will use the first entry in each of those files, and explain their format, so the implied validation step would be to have a look at their contents, and see if they match that. I can re-state that and give an example below, to make it more explicit.